src/main/java/com/takipi/oss/storage/caching/CacheLogger.java

```diff
@@ -0,0 +1,88 @@
+package com.takipi.oss.storage.caching;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+import com.takipi.oss.storage.helper.StringUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CacheLogger extends CacheDelegator {
+    private static final Logger logger = LoggerFactory.getLogger("backendcache");
```

Comment:
We used "backendcache" in order to distinct the cache logger from other loggers.
This doesn't apply to `takipi-storage` since we don't have separate log files for different loggers.
Please change it to CacheLogger.class just like the other loggers